src/app/personside/infotabs/saksoversikt/DokumentKomponent.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import { DokumentMetadata as DokumentInterface, Entitet } from '../../../../models/saksoversikt/dokumentmetadata';
+import styled from 'styled-components';
+import theme from '../../../../styles/personOversiktTheme';
+import * as moment from 'moment';
+import { saksdatoSomDate } from '../../../../models/saksoversikt/fellesSak';
+import { UnmountClosed } from 'react-collapse';
+import { getSaksdokument } from '../../../../utils/url-utils';
+import { Element, Normaltekst } from 'nav-frontend-typografi';
+import Dokument from '../../../../svg/Dokument';
+import { AlertStripeAdvarsel } from 'nav-frontend-alertstriper';
+import DokumentIkkeTilgangMerket from '../../../../svg/DokumentIkkeTilgangMerket';
+
+interface Props {
+    dokument: DokumentInterface;
+    harTilgang: boolean;
+}
+
+interface State {
+    åpnet: boolean;
+}
+
+const Wrapper = styled<{ åpen: boolean }, 'div'>('div')`
+  ${props => props.åpen && 'background-color: rgba(0, 0, 0, 0.03);'}
+`;
+
+const InfoWrapper = styled.div`
+  padding: ${theme.margin.px20} ${theme.margin.px10};
+  display: flex;
+  svg {
+    padding-right: 1rem;
+    height: ${theme.margin.px30};
+    width: auto;
+  }
+  cursor: pointer;
+  &:hover {
+    background-color: ${theme.color.objektlisteHover};
+  }
+  &:active {
+    background-color: ${theme.color.objektlisteActive};
+  }
+`;
+
+const EmbedWrapper = styled.div`
+  padding: 1rem;
+  object {
+    box-shadow: 0 0 2rem #888;
+  }
+`;
+
+function formaterEntitet(fra: Entitet) {
+    if (fra === Entitet.Nav) {
+        return 'NAV';
+    }
+    if (fra === Entitet.Sluttbruker) {
+        return 'bruker';
+    }
+    return 'andre';
+}
+
+function formaterDatoOgAvsender(date: Date, fra: Entitet) {
+    return `${moment(date).format('DD.MM.YYYY')} / Fra ${formaterEntitet(fra)}`;
+}
+
+function dokumentIkon(harTilgang: boolean) {
+    if (harTilgang) {
+        return <Dokument/>;
+    } else {
+        return <DokumentIkkeTilgangMerket/>;
+    }
+}
+
+function visDokument(harTilgang: boolean, url: string) {
+    if (harTilgang) {
+        return <object data={url} width={'100%'} height={'500px'}/>;
+    } else {
+        return <AlertStripeAdvarsel>Du har ikke tilgang til dokument.</AlertStripeAdvarsel>;
+    }
+}
+
+class DokumentKomponent extends React.Component<Props, State> {
+    constructor(props: Props) {
+        super(props);
+        this.state = {
+            åpnet: false
+        };
+    }
+
+    toggle() {
+        const åpnet = !this.state.åpnet;
+        this.setState({åpnet: åpnet});
+    }
+
+    render() {
+        const dokument = this.props.dokument;
+        const saksid = dokument.tilhørendeFagsaksid ? dokument.tilhørendeFagsaksid : dokument.tilhørendeSaksid;
+        const dokUrl = getSaksdokument(dokument.journalpostId, dokument.hoveddokument.dokumentreferanse);
+
+        return (
+            <Wrapper åpen={this.state.åpnet} onClick={() => this.toggle()}>
```

Comment:
Det må lages en knapp for dette også, onClick på div funker dårlig for skjermlesere

Reply:
Har laget knappen, men sliter med å plassere den. Er noe fundamentalt med layout jeg ikke skjønner. tror vi må ta et layout-kurs (hint hint). Kan du se på dette og se om du kan reformattere litt så den blir liggende pent? Skal se på de andre tingene etterhvert.